Dockerfile.template

```diff
@@ -28,13 +28,17 @@ RUN apk add --update -t build-deps build-base git zlib-dev openssl-dev \
  && rm -rf /usr/src/*
 
 # Mirror Websites
-ENV SITES https://5pi.de
+ENV SITES http://elpissite.weebly.com
 RUN mkdir -p /content/www \
  && cd /content/www \
- && httrack $SITES \
+ && httrack --ext-depth=1 $SITES \
```

Comment:
Do you think we need ext-depth=1? As I understand it, it should still download *everything* below elpissite.weebly.com. I'm worried it might pull in more stuff than we expect.. Then again, it's probably fine, given that we control where we link to.